app/services/gemmies/create.rb

```diff
@@ -12,6 +12,20 @@ def initialize(gemmy)
       end
     end
 
+    class NotFound < Error
+      attr_reader :gemmy_name
```

Comment:
I'd prefer to store `gemmy` here instead of `gemmy_name`, just like `AlreadyExists` does.